e2e/provision/playbooks/roles/bootstrap/tasks/system-setup.yml

```diff
@@ -0,0 +1,23 @@
+## © 2023 Nephio Authors
+## Licensed under the Apache License 2.0
+## SPDX-License-Identifier: Apache-2.0
+---
+- name: systctl config
```

Comment:
It seems like there is an [ansible module](https://docs.ansible.com/ansible/latest/collections/ansible/posix/sysctl_module.html) for managing sysctl entries, so manybe this can be implemented like

```yaml
- name: Set kernel parameters
  ansible.posix.sysctl:
    name: {{ item.name }}"
    value: "{{ item.value }}"
    state: present
  loop:
    - {name: fs.inotify.max_user_watches, value: 524288}
    - {name: fs.inotify.max_user_instances, value:512}
    - {name: kernel.keys.maxkeys, value:500000 }
    - {name: kernel.keys.maxbytes, value:1000000}
```